tardis/visualization/widgets/shell_info.py

```diff
@@ -4,7 +4,8 @@
     atomic_number2element_symbol,
     species_tuple_to_string,
 )
-from tardis.simulation import Simulation
+
+# from tardis.simulation import Simulation
```

Comment:
This Import can be removed without any issues. Ran the `generating_widgets` notebook without any problems 😉

Reply:
Is this relevant to the current PR, though?